1.Exploratory_Data_Analysis/scripts/4.cell_count_analysis.py

```diff
@@ -25,14 +25,14 @@
 
 
 # Set Path
-path = pathlib.Path(f"../data/{celltype}_preprocessed_sc_norm.parquet")
+path = pathlib.Path(f"../../data/{celltype}_preprocessed_sc_norm.parquet")
 
 
 # In[4]:
 
 
 # import data
-df = pq.read_table(path).to_pandas()
+df = pd.read_parquet(path)
```

Comment:
probably ok for EDA and this project, but improving variable naming (to describe what the dataframe holds) will help with clarity in your next project